app/Http/Controllers/SubscribeController.php

```diff
@@ -147,15 +148,19 @@ public function showManage($code = null)
         }
 
         $subscriber = Subscriber::where('verify_code', '=', $code)->first();
+        $usedComponentGroups = Component::enabled()->where('group_id', '>', 0)->groupBy('group_id')->pluck('group_id');
```

Comment:
I have only dealt with laravel briefly in the past; so I apologize in advance for anything that is overly hack-ish :)